release-notes.md

```diff
@@ -50,16 +50,16 @@ This is a big update to FCS. There are significant trimmings and renamings of th
 Renamings:
 
 ```diff
--type FSharp.Compiler.AbstractIL.Internal.Library.IFileSystem
-+type FSharp.Compiler.SourceCodeServices.IFileSystem
+-type FSharp.Compiler.AbstractIL.Library.IFileSystem
```

Comment:
This is only correct if we plan on getting this in for FCS 39, which to my knowledge isn't the case? At least not that I'm aware of

Reply:
I don't mind - just let me know the schedule.  

(TBH I'm ok with delaying 39 until this is in, even if it's not 100% complete it's still better to get the foundational renames into the ecosystem.  But I don't know if there are people desperate to get 39 or not)

Reply:
We need to, at a minimum, release FCS 39 with VS 16.9. VS 16.9 bits are locked down for bug fixes only. So I highly doubt we'd incorporate this right now since it's helpful from a coherency standpoint - until we have a shipping cadence for FCS properly established - to have bits map to one another cleanly.

Reply:
FCS 39 would come with a bunch of bug fixes and improvements already, so folks definitely would want it

Reply:
Yep, eagerly anticipating a drop of 39 (since the nightlies aren't usable for packaging reasons :) ).

Reply:
OK great thank you.  So I think that means we can integrate this to master and not into 16.9 release branch?  I'll make sure the release notes are separate under 40.0